### PR TITLE
Add Json handling using FSharp.SystemTextJson to docs

### PR DESCRIPTION
**Pull request by wilsoncg**
In creating a standalone WebAssembly app (without remoting), I wanted to consume some JSON. Specifically using an anonymous record, as they are lightweight to define & throwaway. 
It seems out of the box, `System.Text.Json` does not quite handle F# record to reference type conversion. Adding `struct` to the `JsonSerializer.Deserialize` doesn't work without some extra machinery. I was almost ready to give up but then noticed @Tarmil had already created the extra machinery and included it by default into Bolero!

I have added some initial information to the docs, which basically copies the excellent notes in the FSharp.SystemTextJson readme. I was not sure whether it was necessary to copy everything from the readme.

I hope this addition can be added, so that it may save others some time who find themselves in a similar situation.

### CODE REVIEW

**Comment by srid** [comment]
This would be useful to have for users who run Bolero in WebAssembly mode without a server (thus there is no remoting). Eg: https://github.com/srid/GitHubUser/blob/8c7e0f7795ba8c198f8cc2f79ec8a518265b6b7d/src/GitHubUser/Main.fs#L13-L18

**Comment by wilsoncg** [comment]
nudge @granicz / @Tarmil 🙏
Can this be merged please or is it no good?

File: src/Website/docs/JSON.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Dealing with JSON
+subtitle: Use the inbuilt extensions to [System.Text.Json](https://github.com/Tarmil/FSharp.SystemTextJson) for handling F# union and records
```
Comment (granicz) [inline src/Website/docs/JSON.md]: Can you make the link to `FSharp.SystemTextJson` here be on the term "extensions" and not `System.Text.Json`? Also, you might want to make "union" in the last bit a plural, or just use the more encompassing but somewhat dull "F#-specific types" instead.

File: src/Website/docs/JSON.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+```fsharp
+JsonSerializer.Serialize({| x = "Hello"; y = "world!" |}, options)
+// --> {"x":"Hello","y":"world!"}
+
+JsonSerializer.Deserialize<{| x : string ; y : string |}>("""{"x":"Hello","y":"World!"}""", options)
+// --> {| x = "Hello"; y = "world!" |}
+```
+
+### More info
+For Unions, Tuples, Options, Maps, Lists & Sets, more information is on the [FSharp.SystemTextJson](https://github.com/Tarmil/FSharp.SystemTextJson) project site.
```
Comment (granicz) [inline src/Website/docs/JSON.md]: Not sure I'd capitalize these.

File: src/Website/docs/JSON.md (hunk 1 above)
Comment (wilsoncg) [reply src/Website/docs/JSON.md]: done

File: src/Website/docs/JSON.md (hunk 2 above)
Comment (wilsoncg) [reply src/Website/docs/JSON.md]: amended

**Comment by wilsoncg** [comment]
Thanks for the feedback @granicz, I have applied your comments.

**Comment by granicz** [comment]
Thanks!

**Pull request merged**